contracts.go

```diff
@@ -0,0 +1,127 @@
+package emulator
+
+import (
+	"fmt"
+
+	flowsdk "github.com/onflow/flow-go-sdk"
+	"github.com/onflow/flow-go-sdk/templates"
+	"github.com/onflow/flow-go/fvm"
+	flowgo "github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-nft/lib/go/contracts"
+	fusd "github.com/onflow/fusd/lib/go/contracts"
+	nftstorefront "github.com/onflow/nft-storefront/lib/go/contracts"
+)
+
+var CommonContracts = func() []ContractDescription {
```

Comment:
NIT: I think this could be a variable not a function. But I'm not sure if better since then you could modify it.

Reply:
I am new to Golang. Could I have a global variable definition, with a body? Because I need the chain and service address intermediate variables.

Reply:
I personally like the function version, because all the intermediate variables, are not exposed to the rest of the file :innocent: